### DISCUSSION TITLE
Implementing count into custom keybinds/functions

### DISCUSSION THREAD

**Discussion by ghost** [qutebrowser/qutebrowser#6689] [Ideas] [answered]
I would like to be able to access `count` the way that built-in commands like `tab-move` are able to. For example, I am able to type `2gm` to move my current tab to the second index. I would like to say start an mpv playlist at a specific count when I specify a count. i.e. `3,p` would equate to `mpv --playlist-start=[count].` Not sure if this is a feature but could not find any documentation on how to access this count in custom scripts/keybinds.

**Comment by The-Compiler** [comment, marked as answer]
You can write a [userscript](https://github.com/qutebrowser/qutebrowser/blob/master/doc/userscripts.asciidoc) and use `$QUTE_COUNT` there.
